### PR TITLE
rootless: copy some settings from the global configuration

### PR DESCRIPTION
**Pull request by giuseppe**
if some paths are overriden in the global configuration file, be sure
that rootless podman honors them.

Closes: https://github.com/containers/libpod/issues/2174

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
Code LGTM

File: libpod/runtime.go [hunk 1]
```diff
@@ -385,6 +385,26 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
...
+				// Ignore any error, the file might not be readable by us.
+				continue
+			}
+			tmpConfig := new(RuntimeConfig)
+			if _, err := toml.Decode(string(contents), tmpConfig); err != nil {
+				return nil, errors.Wrapf(err, "error decoding configuration file %s", path)
+			}
+
+			// Cherry pick the settings we want from the global configuration
+			runtime.config.ConmonPath = tmpConfig.ConmonPath
```
Comment (rhatdan) [inline libpod/runtime.go]: Shouldn't we make sure these are not set, and only set them.

If I set the ConmonPath in my local config isn't this an issue.

Comment (giuseppe) [reply libpod/runtime.go]: We are in this code branch only when `~/.config/containers/libpod.conf` doesn't exist.

The only place where we could set them is from the default configuration we create.

**Comment by giuseppe** [comment]
let's wait for @sysrich before merging this

File: libpod/runtime.go (hunk 1 above)
Comment (rhatdan) [reply libpod/runtime.go]: Ok then LGTM

**Comment by giuseppe** [comment]
rebased, tests are green now

**Pull request merged**